config/config.js

```diff
@@ -81,19 +81,23 @@ exports.config = {
     disabled  : true
   },
 
-  /*
-   * PS3 support is supplied by Gimx and is available on Windows and Linux
-   * only.  Refer to the README for information on setup.
-   */
   ps3 : {
+    deviceNotes: [
```

Comment:
Although I don't like documentation in the JS object, it's likely unavoidable.  Once that's done and normal JS comments removed, we can move this file to proper .json (I had only left it .js for linters complaining about comments).